taskcat/client_factory.py

```diff
@@ -139,6 +143,8 @@ def get(self, service, region=None, credential_set='default', aws_access_key_id=
             )
             if credential_set not in self._credential_sets.keys():
                 raise KeyError('credential set %s does not exist' % credential_set)
+            if region:
```

Comment:
shouldn't this be:
```
if region and region in self._credential_sets.keys():
```
so that you can specify `region` without needing to specify `regional_cred_map`

Reply:
Good catch, my bad.